.github/workflows/_python-tests.yml

```diff
@@ -112,6 +112,23 @@ jobs:
         shell: bash
         env:
           PROPERTY_TESTING_PRESET: ${{ inputs.property_testing_preset }}
+      - name: Make get logs script executable
+        if: success() || failure()
```

Comment:
What is this if not success or failure?

Reply:
This is to ensure that logs get collected and uploaded even if the test fails or succeeds. Without this, the logs don't get collected when the test fails